? .mozconfig.mk
? .mozconfig.out
? bug176182-jsautorequest-sb_0_2a_branch.patch
? bug204114-unicharutils-sb_0_2a_branch.patch
? bug280236-classinfo_v1.6-sb_0_2a_branch.patch
? bug330640-accessibility-sb_0_2a_branch.patch
? bug335549-uuidgen-sb_0_2a_branch.patch
? bug344439-setInterval-linux64-sb_0_2a_branch.patch
? bug345541-pluginloading-sb_0_2a_branch.patch
? bug346843-sighandler-x86_64-sb_0_2a_branch.patch
? bug347287-ahcrap64-sb_0_2a_branch.patch
? js_gclogging-sb_0_2a_branch.patch
? js_tracerefcnt-sb_0_2a_branch.patch
? sanity.diff
Index: xpfe/bootstrap/nsSigHandlers.cpp
===================================================================
RCS file: /cvsroot/mozilla/xpfe/bootstrap/nsSigHandlers.cpp,v
retrieving revision 1.41.18.1
diff -u -p -r1.41.18.1 nsSigHandlers.cpp
--- xpfe/bootstrap/nsSigHandlers.cpp	22 Jun 2006 22:31:13 -0000	1.41.18.1
+++ xpfe/bootstrap/nsSigHandlers.cpp	30 Aug 2006 22:52:56 -0000
@@ -76,7 +76,7 @@
 static char _progname[1024] = "huh?";
 static unsigned int _gdb_sleep_duration = 300;
 
-#if defined(LINUX) && defined(DEBUG) && (defined(__i386) || defined(PPC))
+#if defined(LINUX) && defined(DEBUG) && (defined(__i386) || defined(__x86_64) || defined(PPC))
 #define CRAWL_STACK_ON_SIGSEGV
 #endif
  
